third_party/move/move-vm/test-utils/src/storage.rs

```diff
@@ -158,7 +165,7 @@ impl<'a, 'b, S: MoveResolver> DeltaStorage<'a, 'b, S> {
 /// Simple in-memory storage for modules and resources under an account.
 #[derive(Debug, Clone)]
 struct InMemoryAccountStorage {
-    resources: BTreeMap<StructTag, Bytes>,
+    resources: BTreeMap<StructTag, (Bytes, Option<MoveTypeLayout>)>,
```

Comment:
Do we use layouts anywhere here? If not, maybe we can not have it at all

Reply:
In `bad_storage_tests.rs` line 108, we apply ChangeSet outputted by session to InMemoryStorage. So, InMemoryStorage had to take Changes<Bytes, (Bytes, Option<MoveTypeLayout>)> as input.